Configuration/NodeTypes.Overrides.yaml

```diff
@@ -0,0 +1,9 @@
+'Neos.Neos:Node':
+  options:
+    fusion:
+      prototypeGenerator: ~
```

Comment:
this is a rather controversial "best practice"

Reply:
This is not needed any more since with 4.0 this is the default in the core ;-) for Node and Document